src/client/src/rt-theme/themes.ts

```diff
@@ -1,53 +1,100 @@
 import _ from 'lodash'
 
-import { colors, Palette, PaletteMap } from './colors'
-
-export type Theme = typeof defaultTheme &
-  ColorPair & {
-    primary: Palette
-    secondary: Palette
-    accents: PaletteMap
+import { AccentPaletteMap, Color, ColorPaletteMaps, CorePalette, CorePaletteMap } from './colors'
+import { colors, dark, light } from './colors'
+
+export interface Theme {
+  primary: CorePalette
+  secondary: CorePalette
+  accents: AccentPaletteMap
+  colors: ColorPaletteMaps
+
+  motion: Motion & {
+    fase: Motion
+    normal: Motion
+    slow: Motion
   }
 
-interface ColorPair {
-  backgroundColor: string
-  textColor?: string
+  shell: ColorPair
+  component: ColorPair
+
+  button: TouchableIntents
+
+  // Known extensible properties
+  backgroundColor?: Color
+  textColor?: Color
+
+  // TODO: remove after theme migration
```

Comment:
Can we date and initial any checked in TODO's plz

Reply:
How has this never occurred to me?? 

Yes, definitely.